packages/models/src/store/store.ts

```diff
@@ -77,15 +78,24 @@ export class Store<
       options?: Option
     },
   ) {
-    super()
-    this.cellPattern = Reflect.getMetadata(ProviderKey.CellPattern, this.constructor) || params?.cellPattern
+    super(ref)
+    const resourceBindingRegister = Reflect.getMetadata(
+      ProviderKey.ResourceBindingRegister,
+      this.constructor,
+      this.ref.uri,
+    )
+    if (resourceBindingRegister) {
+      this.call('local://resource', resourceBindingRegister)
```

Comment:
`this.call('local://resource', resourceBindingRegister)` maybe can't call `Manager.handleCall`. 
Because the manager is registered in the `mvp-dapp`https://github.com/ckb-js/kuai/blob/9f2da62f1a799ffffce72d009deaed169b6cb0ce/packages/samples/mvp-dapp/src/main.ts#L52. If you should ensure it, you should register `manager` in the framework.

Reply:
> `this.call('local://resource', resourceBindingRegister)` maybe can't call `Manager.handleCall`. Because the manager is registered in the `mvp-dapp`
> 
> https://github.com/ckb-js/kuai/blob/9f2da62f1a799ffffce72d009deaed169b6cb0ce/packages/samples/mvp-dapp/src/main.ts#L52
> 
> . If you should ensure it, you should register `manager` in the framework.

I will use the `ActorProvider` decorator to register the resource binding manager